src/app/ContentExportDialog.qml

```diff
@@ -0,0 +1,120 @@
+/*
```

Comment:
I'm a bit concerned with the size of this PR, both in commits and lines. Is it possible to resubmit it as multiple PRs, each one making a change which can stand without the others?

As it is, I would have difficulty merging this PR without making the history confusing to read for anyone trying to trace issues in the future.

Reply:
I'll check if I can separate them into smaller PRs but I may only be able to do so after two weeks 😅 But I'll try to do it soon as I can.